chat/message/src/Message/Message.stories.tsx

```diff
@@ -130,43 +134,97 @@ VerifiedAnswer.args = {
     verifier: 'MongoDB Staff',
     verifiedAt: new Date('2023-08-24T16:20:00Z'),
   },
-  // @ts-ignore onChange is passed in the story itself
-  children: <MessageFeedbackStory />,
+  children: <MessageFeedback />,
+};
+
+function useCorrectBaseFontSize() {
```

Comment:
We also have a [util](https://github.com/mongodb/leafygreen-ui/blob/main/packages/typography/src/utils/useUpdatedBaseFontSize.ts) to help with this.